pkgs/by-name/lo/lovely-injector/package.nix

```
@@ -4,20 +4,19 @@
   lib,
 }:
 let
-  version = "0.6.0";
-  lovelyInjector = fetchFromGitHub {
-    owner = "vgskye";
-    repo = "lovely-injector";
-    rev = "3224915f4d47b557c34b5012797cf92d4cc629af";
-    hash = "sha256-fzkuuu6pmvqeJa7qlX8jhtCLC4oYRLUm1hqHTRiYEX8=";
-  };
+  version = "0.7.1";
 in
 rustPlatform.buildRustPackage rec {
   pname = "lovely-injector";
   inherit version;
-  src = lovelyInjector;
+  src = fetchFromGitHub {
+    owner = "ethangreen-dev";
+    repo = "lovely-injector";
+    hash = "sha256-fzkuuu6pmvqeJa7qlX8jhtCLC4oYRLUm1hqHTRiYEX8=";
+    tag = "v${version}";
+  };
   useFetchCargoVendor = true;
-  cargoHash = "sha256-Mkmj+ENdUge1V1cVAQOV2K01sYKEyhxTse0f5o6H6Xc=";
+  cargoHash = "sha256-hHq26kSKcqEldxUb6bn1laTpKGFplP9/2uogsal8T5A=";
```

```suggestion
  cargoHash = "sha256-Mkmj+ENdUge1V1cVAQOV2K01sYKEyhxTse0f5o6H6Xc=";
```
See https://github.com/NixOS/nixpkgs/pull/385875/checks?check_run_id=38385609253